misk-hibernate/src/main/kotlin/misk/hibernate/DbChild.kt

```diff
@@ -0,0 +1,10 @@
+package misk.hibernate
+
+/**
+ * Marker interface for persistent entities. Ensures that only persistent entities can be passed
```

Comment:
nit: comment is duplicative with `DbMember`. Probably want to specify a comment more specific to `DbChild` and its relationship with `DbRoot` and other entities.

Reply:
comments updated